apps/examples/testcase/le_tc/kernel/tc_debug.c

```diff
@@ -0,0 +1,104 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_debug.c
+
+/// @brief Test Case Example for Errno API
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include "tc_internal.h"
+
+#define DEVPATH "/dev/sysdbg"
+#define BUFLEN 32
+
+static void tc_debug_sysdbg_write(void);
+
+#ifdef CONFIG_TC_KERNEL_DEBUG
+/**
+ * @fn                  :tc_debug_sysdbg_write
+ * @brief               :fops to read data buffer passed from user space
```

Comment:
I think tc writes the data, not read data from buffer.
Please check the description.

Reply:
please check sysdbg_write() in sysdbg.c